README.md

```diff
@@ -1,207 +1,143 @@
+Run the following to install e.g. version `5.0.1`:
 
 ```
-$ hz mc start -J <option1>,<option2>
-``` 
-
-You can use `,` to separate multiple options. Please note that class path settings (such as `-cp`, `-jar`) are **not** allowed. 
+brew install hazelcast@5.0.1
+```
 
-When you run `hz mc start` command with any available option, it starts a Hazelcast Management Center instance in the foreground. Please use Ctrl+C (SIGINT) to gracefully stop the running instance. 
+## Installing a snapshot version
 
-For all other available options, refer to `hz mc start --help`. 
+### Install a snapshot version with apt
 
-### How to print version information
+You need to replace `stable` with `snapshot` in the repository
+definition to use Hazelcast snapshots.
 
-This command prints Hazelcast, Management Center and the tool's version information to the console output: 
+Run the following to install the latest snapshot version:
 
 ```
-$ hz -V
-CLI tool: 4.2020.12
-Hazelcast: 4.1.1
-Hazelcast Management Center: 4.2020.12
+wget -qO - https://repository.hazelcast.com/api/gpg/key/public | sudo apt-key add -
+echo "deb https://repository.hazelcast.com/debian snapshot main" | sudo tee -a /etc/apt/sources.list
+sudo apt update && sudo apt install hazelcast
 ```
 
-## Installation from source
-
-Firstly, build the distribution:
+### Install a snapshot version with yum
 
-```
-cd distro/
-make
-```
+You need to replace `stable` with `snapshot` in the repository
+definition to use Hazelcast snapshots.
 
-After running the `make` command successfully, navigate to `distro/build/dist/hazelcast-{hazelcast.version}/bin` folder and run the `./hz` script:  
+Run the following to install the latest snapshot version:
 
 ```
-cd build/dist/hazelcast-{hazelcast.version}/bin
-./hz
+wget https://repository.hazelcast.com/rpm/snapshot/hazelcast-rpm.repo -O hazelcast-snapshot-rpm.repo
+sudo mv hazelcast-snapshot-rpm.repo /etc/yum.repos.d/
+sudo yum install hazelcast
 ```
 
-This will print the usage options. You can follow the instructions in the output.
-
-The compressed distribution package is also available at `distro/build/dist/` folder. You can extract & use it.
+### Install a snapshot version with Homebrew
 
+You need to add `.snapshot` suffix to the package version when 
+installing a snapshot. 
 
-### Checkstyle validation
-
-Please use the following command to run checkstyle validation:
+Run the following to install the latest `5.1-SNAPSHOT` version:
 
 ```
-mvn -P checkstyle clean validate
+brew tap hazelcast/hz
+brew install hazelcast@5.1.snapshot
 ```
 
-### SpotBugs analysis
+## Running Hazelcast
 
-Please use the following command to run [SpotBugs](https://spotbugs.github.io/) analysis:
+After successful installation all commands from Hazelcast distribution
+`bin` directory should be on path.
 
-```
-mvn -P spotbugs clean compile
-```
+Run the following command to start a Hazelcast server with the default configuration:
```

Comment:
What do you think about mentioning that additional options can be checked with `hz start --help`?

Reply:
Added.